src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2685,9 +2680,11 @@ namespace EnergyPlus::ZoneEquipmentManager {
         // and controllers
 
         if (ZoneAirMassFlow.EnforceZoneMassBalance) {
-            if (ZoneAirMassFlow.ZoneFlowAdjustment == DataHeatBalance::NoAdjustReturnAndMixing) AdjustZoneMixingFlowFlag = false;
-            if (ZoneAirMassFlow.ZoneFlowAdjustment == DataHeatBalance::NoInfiltrationFlow) AdjustZoneInfiltrationFlowFlag = false;
-            CalcAirFlowSimple(state, 0, AdjustZoneMixingFlowFlag, AdjustZoneInfiltrationFlowFlag);
+            if (FirstHVACIteration) {
+                CalcAirFlowSimple(state, 0);
+            } else {
+                CalcAirFlowSimple(state, 0, ZoneAirMassFlow.AdjustZoneMixingFlow, ZoneAirMassFlow.AdjustZoneInfiltrationFlow);
+            }
```

Comment:
Adjust the zone mixing and infiltration air flows only when "FirstHVACIteration" is false.